website/src/pages/roadmap.md

```diff
@@ -8,64 +8,68 @@ Hudi community strives to deliver major releases every 3-4 months, while offerin
 This page captures the forward-looking roadmap of ongoing & upcoming projects and when they are expected to land, broken
 down by areas on our [stack](blog/2021/07/21/streaming-data-lake-platform/#hudi-stack).
 
+## Recent Release
+[0.15.0](https://issues.apache.org/jira/projects/HUDI/versions/12353381) (June 2024)
```

Comment:
Should we use the release link in the website, i.e., https://hudi.apache.org/releases/release-0.15.0?